content/en/docs/concepts/configuration/overview.md

```diff
@@ -73,17 +73,19 @@ A desired state of an object is described by a Deployment, and if changes to tha
 
 ## Container Images
 
-The [imagePullPolicy](/docs/concepts/containers/images/#updating-images) and the tag of the image affect when the [kubelet](/docs/reference/command-line-tools-reference/kubelet/) attempts to pull the specified image.
+The [imagePullPolicy](/docs/concepts/containers/images/#updating-images) and the tag of the image affect when the [kubelet](/docs/reference/command-line-tools-reference/kubelet/) attempts to pull the specified image. For the most reliable results, always set `imagePullPolicy` explicitly, and do not rely on the default behavior.
```

Comment:
The word default has specific meaning, and the `imagePullPolicy` field doesn't have a default behavior (it has a default value of unset, but the behaviour in this case is complex). Because I think “default” could mislead, I suggest;

```suggestion
The [imagePullPolicy](/docs/concepts/containers/images/#updating-images) and the tag of the image affect when the [kubelet](/docs/reference/command-line-tools-reference/kubelet/) attempts to pull the specified image. For the most reliable results, always set `imagePullPolicy` explicitly (do not rely on the kubelet inferring an image pull policy).
```

Reply:
My _understanding_ (which may be wrong!) was that the Kubelet isn't inferring an image pull policy - rather, the apiserver infers the pull policy _and writes it_ in the container spec. E.g. if you're submitting a Deployment and the container has no `imagePullPolicy`, if you do a `kubectl get -oyaml` on the Deployment immediately after submitting it, you'll see that `imagePullPolicy` has been set.

Could we change it to "(do not rely on Kubernetes inferring an image pull policy)"?

Reply:
That's useful to know. I hadn't checked on the mechanism but if the field is set on admission then I would:
- document that admission-time defaulting
- hyperlink to signpost interest readers to read that detail

(does this mean that if you _change_ the image for a Deployment, the imagePullPolicy set at admission time stays in place regardless - if so, let's document that a few ways too).

_It's OK to log an issue that describes what's missing and rely on the existence of that issue to explain why some PR (eg this one) isn't covering all the detail needed._